modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -192,8 +192,7 @@ docker-compose logs --timestamp owncloud
 
 === Docker Compose YAML File
 
-NOTE: If you are an enterprise customer and are already registered on portal.owncloud.com, replace `image: owncloud/server` with `image: registry.owncloud.com/owncloud/enterprise` to be able to download our enterprise docker image. 
-Then, login to our registry by running `docker login registry.owncloud.com`, along with your portal credentials. 
+NOTE: With 10.5 the `image: registry.owncloud.com/owncloud/enterprise` is deprecated. All supported enterprise features and apps are now included in the public `image: owncloud/server`. A login to our registry `docker login registry.owncloud.com` is no longer needed. 
```

Comment:
Please say something like "Since **ownCloud Server 10.5**, the ..." and mention the relevant release notes entry https://doc.owncloud.com/server/admin_manual/release_notes.html#changes-in-delivery.

Reply:
Please use `xref:<path to file>/release_notes.adoc#changes-in-delivery[link text]`
and not a http link!